### ISSUE TITLE
Upgrade to node 16

### ISSUE THREAD

**Issue by ThorstenWitzig** [mAAdhaTTah/combine-dependabot-prs#18]
Github is [sunsetting node 12](https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/). Would be nice if this cool action would upgrade dependencies that it uses node 16.

**Comment by mAAdhaTTah** [comment]
Yeah, that would be great. Would you be able to open a PR for that?
